LuaUI/Widgets/cmd_distIcon.lua

```diff
@@ -1,22 +0,0 @@
-function widget:GetInfo()
-	return {
-		name = "1944 Icon Distance",
-		desc = "Sets Icon Distance to a suitable level for Spring: 1944",
-		author = "Craig Lawrence",
-		date = "09-12-2008",
-		license = "Public Domain",
-		layer = 1,
-		enabled = true
-	}
-end
-
-local unitIconDist = 0
-
-function widget:Initialize()
-	unitIconDist = Spring.GetConfigInt('UnitIconDist')
-	Spring.SendCommands("disticon 250")

```

Comment:
I see you are now dealing with the icon distance as a GUI option. I like that!
The problem I see is that if you play another game, and `UnitIconDist` is changed, then s44 parameter will be affected as well, right?

Reply:
If I understand the case you talk about, I think this does not happen. UIs are saving the configuration in `SpringData/LuaUI/Configs/<gameShortName>.lua` so the configuration is shared only across multiple versions of same game, not across multiple games. 

New UI is setting the default on 100 I think and then loads it from config. You can manually set your own value in `menu > settings > view > icon distance (slider)` and your custom value will be used from that moment for any other match.

This is my intention, I tested it it works this way in the most common user paths, but I admit it was not tested on all edgecases (like lobby override by starting param, other custom widgets...).

Also the current default is set roughly, just to have something smaller than 250. All the UI tweaks should be done in next iteration already in baseline. In change you comment I just removed the biggest blocker, that widget override setting it to crazy high value.

Btw regarding the default - whats good? Our experience from nota is 80-110, depends on player - and nota have still bigger avg size of unit. So I think while tweaking it for s44, going under 100 should be no problem for s44. From my experience on tradeshows people considered the icon view as the cool feature of whole game (combined with unlimited zoom-out) so this feedback supports us in this direction. Also I personally see no point to show model of the unit in distance from which the type of the unit is not visually clear. Anything above 100 is such situation.

Reply:
> If I understand the case you talk about, I think this does not happen. UIs are saving the configuration in SpringData/LuaUI/Configs/<gameShortName>.lua so the configuration is shared only across multiple versions of same game, not across multiple games.

Are you positive about that? Because i detect the following in epicmenu:

```
LuaUI/Widgets/gui_notAchili_epicmenu.lua:                                       Spring.SetConfigInt( option.springsetting, BoolToInt(option.value) )
LuaUI/Widgets/gui_notAchili_epicmenu.lua:                                               Spring.SetConfigInt( option.springsetting, option.value )
```

TODO: Check it is not overriding global settings

> Btw regarding the default - whats good? Our experience from nota is 80-110, depends on player - and nota have still bigger avg size of unit. So I think while tweaking it for s44, going under 100 should be no problem for s44. From my experience on tradeshows people considered the icon view as the cool feature of whole game (combined with unlimited zoom-out) so this feedback supports us in this direction. Also I personally see no point to show model of the unit in distance from which the type of the unit is not visually clear. Anything above 100 is such situation for Infantry and there it matters the most imo.

No idea at all... I actually just edit it to make snapshots and so on... Anyway, it can be tweaked later...

Reply:
Commands you mentioned does not affect my global settings in lobby, but maybe my lobby is using own way to defend its global settings :)

I have not experienced any transfer of the settings while switching between multiple games. I would keep it open and react based on players feedback. The question is complex anyway - I mean its hard to prevent all possible problems while you and me are using just limited subset of lobby setups, lobby types, autohost setups and personal settings combinations possible while adding such huge new thing.

Do you agree to solve it later if anyone raise the issue?

Reply:
I'm agree!